### ISSUE TITLE
[Transporter Error Output]: An exception has occurred: Broken pipe (Write failed)

### ISSUE THREAD

**Issue by benasher44** [fastlane/fastlane#16749]
🔑
### New Issue Checklist

- [x] Updated fastlane to the latest version
- [x] I read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)
- [x] I read [docs.fastlane.tools](https://docs.fastlane.tools)
- [x] I searched for [existing GitHub issues](https://github.com/fastlane/fastlane/issues)

### Issue Description
Hi there! We've been following along with all of the recent updates, and everything was up and running great with 2.150.0 and 2.150.1. Then sometime on Friday, we started seeing these connection reset / broken pipe issues, which are still ongoing. It's unclear to me if it's an App Store issue, a fastlane issue, or maybe a unique combination :).

Any advice is appreciated — thanks in advance!

##### Command executed
upload_to_app_store(
    ipa: './PlanGrid.ipa',
    force: true,
    skip_screenshots: true,
    skip_app_version_update: false,
    run_precheck_before_submit: false
)

##### Complete output when running fastlane, including the stack trace and command used

<details>
  <pre>[23:20:00]: ---------------------------------
[23:20:00]: --- Step: upload_to_app_store ---
[23:20:00]: ---------------------------------
[23:20:00]: Successfully loaded '/private/var/lib/jenkins/plangrid-adsk/fastlane/Deliverfile' 📄

+------------------+----------------------------+
| Detected Values from './fastlane/Deliverfile' |
+------------------+----------------------------+
| app_identifier   | com.loupe.PlanGrid         |
| username         | <scrubbed> |
| team_name        | PlanGrid, Inc.             |
+------------------+----------------------------+

[23:20:00]: Login to App Store Connect (<scrubbed>)
[23:20:02]: Login successful

+--------------------------------------+----------------------------+
|                      deliver 2.150.1 Summary                      |
+--------------------------------------+----------------------------+
| ipa                                  | ./PlanGrid.ipa             |
| force                                | true                       |
| skip_screenshots                     | true                       |
| skip_app_version_update              | false                      |
| run_precheck_before_submit           | false                      |
| screenshots_path                     | ./fastlane/screenshots     |
| metadata_path                        | ./fastlane/metadata        |
| app_version                          | 5.86.1                     |
| username                             | <scrubbed> |
| app_identifier                       | com.loupe.PlanGrid         |
| platform                             | ios                        |
| edit_live                            | false                      |
| use_live_version                     | false                      |
| skip_binary_upload                   | false                      |
| skip_metadata                        | false                      |
| overwrite_screenshots                | false                      |
| submit_for_review                    | false                      |
| reject_if_possible                   | false                      |
| automatic_release                    | false                      |
| phased_release                       | false                      |
| reset_ratings                        | false                      |
| team_name                            | PlanGrid, Inc.             |
| precheck_default_rule_level          | warn                       |
| individual_metadata_items            | []                         |
| ignore_language_directory_validation | false                      |
| precheck_include_in_app_purchases    | true                       |
+--------------------------------------+----------------------------+

[23:20:02]: Making sure the latest version on App Store Connect matches '5.86.1'...
[23:20:04]: '5.86.1' is the latest version on App Store Connect
[23:20:04]: Loading './fastlane/metadata/en-AU/description.txt'...
[23:20:04]: Loading './fastlane/metadata/en-AU/keywords.txt'...
[23:20:04]: Loading './fastlane/metadata/en-AU/release_notes.txt'...
[23:20:04]: Loading './fastlane/metadata/en-AU/support_url.txt'...
[23:20:04]: Loading './fastlane/metadata/en-AU/marketing_url.txt'...
[23:20:04]: Loading './fastlane/metadata/en-AU/promotional_text.txt'...
[23:20:04]: Loading './fastlane/metadata/en-AU/name.txt'...
[23:20:04]: Loading './fastlane/metadata/en-AU/subtitle.txt'...
[23:20:04]: Loading './fastlane/metadata/en-AU/privacy_url.txt'...
[23:20:04]: Loading './fastlane/metadata/en-CA/description.txt'...
[23:20:04]: Loading './fastlane/metadata/en-CA/keywords.txt'...
[23:20:04]: Loading './fastlane/metadata/en-CA/release_notes.txt'...
[23:20:04]: Loading './fastlane/metadata/en-CA/support_url.txt'...
[23:20:04]: Loading './fastlane/metadata/en-CA/marketing_url.txt'...
[23:20:04]: Loading './fastlane/metadata/en-CA/promotional_text.txt'...
[23:20:04]: Loading './fastlane/metadata/en-CA/name.txt'...
[23:20:04]: Loading './fastlane/metadata/en-CA/subtitle.txt'...
[23:20:04]: Loading './fastlane/metadata/en-CA/privacy_url.txt'...
[23:20:04]: Loading './fastlane/metadata/en-GB/description.txt'...
[23:20:04]: Loading './fastlane/metadata/en-GB/keywords.txt'...
[23:20:04]: Loading './fastlane/metadata/en-GB/release_notes.txt'...
[23:20:04]: Loading './fastlane/metadata/en-GB/support_url.txt'...
[23:20:04]: Loading './fastlane/metadata/en-GB/marketing_url.txt'...
[23:20:04]: Loading './fastlane/metadata/en-GB/promotional_text.txt'...
[23:20:04]: Loading './fastlane/metadata/en-GB/name.txt'...
[23:20:04]: Loading './fastlane/metadata/en-GB/subtitle.txt'...
[23:20:04]: Loading './fastlane/metadata/en-GB/privacy_url.txt'...
[23:20:04]: Loading './fastlane/metadata/en-US/description.txt'...
[23:20:04]: Loading './fastlane/metadata/en-US/keywords.txt'...
[23:20:04]: Loading './fastlane/metadata/en-US/release_notes.txt'...
[23:20:04]: Loading './fastlane/metadata/en-US/support_url.txt'...
[23:20:04]: Loading './fastlane/metadata/en-US/marketing_url.txt'...
[23:20:04]: Loading './fastlane/metadata/en-US/promotional_text.txt'...
[23:20:04]: Loading './fastlane/metadata/en-US/name.txt'...
[23:20:04]: Loading './fastlane/metadata/en-US/subtitle.txt'...
[23:20:04]: Loading './fastlane/metadata/en-US/privacy_url.txt'...
[23:20:04]: Loading './fastlane/metadata/es-ES/description.txt'...
[23:20:04]: Loading './fastlane/metadata/es-ES/keywords.txt'...
[23:20:04]: Loading './fastlane/metadata/es-ES/release_notes.txt'...
[23:20:04]: Loading './fastlane/metadata/es-ES/support_url.txt'...
[23:20:04]: Loading './fastlane/metadata/es-ES/marketing_url.txt'...
[23:20:04]: Loading './fastlane/metadata/es-ES/promotional_text.txt'...
[23:20:04]: Loading './fastlane/metadata/es-ES/name.txt'...
[23:20:04]: Loading './fastlane/metadata/es-ES/subtitle.txt'...
[23:20:04]: Loading './fastlane/metadata/es-ES/privacy_url.txt'...
[23:20:04]: Loading './fastlane/metadata/es-MX/description.txt'...
[23:20:04]: Loading './fastlane/metadata/es-MX/keywords.txt'...
[23:20:04]: Loading './fastlane/metadata/es-MX/release_notes.txt'...
[23:20:04]: Loading './fastlane/metadata/es-MX/support_url.txt'...
[23:20:04]: Loading './fastlane/metadata/es-MX/marketing_url.txt'...
[23:20:04]: Loading './fastlane/metadata/es-MX/promotional_text.txt'...
[23:20:04]: Loading './fastlane/metadata/es-MX/name.txt'...
[23:20:04]: Loading './fastlane/metadata/es-MX/subtitle.txt'...
[23:20:04]: Loading './fastlane/metadata/es-MX/privacy_url.txt'...
[23:20:04]: Loading './fastlane/metadata/copyright.txt'...
[23:20:04]: Loading './fastlane/metadata/primary_category.txt'...
[23:20:04]: Loading './fastlane/metadata/secondary_category.txt'...
[23:20:04]: Loading './fastlane/metadata/primary_first_sub_category.txt'...
[23:20:04]: Loading './fastlane/metadata/primary_second_sub_category.txt'...
[23:20:04]: Loading './fastlane/metadata/secondary_first_sub_category.txt'...
[23:20:04]: Loading './fastlane/metadata/secondary_second_sub_category.txt'...
[23:20:04]: Loading './fastlane/metadata/review_information/first_name.txt'...
[23:20:04]: Loading './fastlane/metadata/review_information/last_name.txt'...
[23:20:04]: Loading './fastlane/metadata/review_information/phone_number.txt'...
[23:20:04]: Loading './fastlane/metadata/review_information/email_address.txt'...
[23:20:04]: Loading './fastlane/metadata/review_information/demo_user.txt'...
[23:20:04]: Loading './fastlane/metadata/review_information/demo_password.txt'...
[23:20:04]: Loading './fastlane/metadata/review_information/notes.txt'...
[23:20:45]: Will begin uploading metadata for '5.86.1' on App Store Connect
[23:20:45]: Uploading metadata to App Store Connect for localized version 'es-ES'
[23:20:45]: Uploading metadata to App Store Connect for localized version 'en-AU'
[23:20:46]: Uploading metadata to App Store Connect for localized version 'en-GB'
[23:20:46]: Uploading metadata to App Store Connect for localized version 'es-MX'
[23:21:59]: Uploading metadata to App Store Connect for localized version 'en-CA'
[23:21:59]: Uploading metadata to App Store Connect for localized version 'en-US'
[23:21:59]: Uploading metadata to App Store Connect for localized info 'en-GB'
[23:22:01]: Uploading metadata to App Store Connect for localized info 'en-AU'
[23:22:03]: Uploading metadata to App Store Connect for localized info 'en-CA'
[23:22:05]: Uploading metadata to App Store Connect for localized info 'en-US'
[23:22:07]: Uploading metadata to App Store Connect for localized info 'es-MX'
[23:22:10]: Uploading metadata to App Store Connect for localized info 'es-ES'
[23:22:12]: Uploading metadata to App Store Connect for version
[23:22:12]: Category 'MZGenre.Business' from iTunesConnect as been deprecated. Please replace with 'BUSINESS'
[23:22:12]: Category 'MZGenre.Productivity' from iTunesConnect as been deprecated. Please replace with 'PRODUCTIVITY'
[23:22:12]: You can find more info at https://docs.fastlane.tools/actions/deliver/#reference
[23:22:15]: Uploading app review information to App Store Connect
[23:22:16]: Uploading binary to App Store Connect
[23:22:29]: iTunes Transporter successfully finished its job
[23:22:29]: Going to upload updated app to App Store Connect
[23:22:29]: This might take a few minutes. Please don't interrupt the script.
[23:28:17]: [Transporter Error Output]: An error occurred while uploading the file f43b28f31f34618cac4fe239c056fc057778874e3c78f80d291bd97a50d7e120.ipa.

[23:28:17]: [Transporter Error Output]: An exception has occurred: Broken pipe (Write failed)

[23:28:17]: [Transporter Error Output]: Failed to upload file: /tmp/498795789.itmsp/f43b28f31f34618cac4fe239c056fc057778874e3c78f80d291bd97a50d7e120.ipa

[23:30:40]: Transporter transfer failed.
[23:30:40]: 
[23:30:40]: An error occurred while uploading the file f43b28f31f34618cac4fe239c056fc057778874e3c78f80d291bd97a50d7e120.ipa.
An exception has occurred: Broken pipe (Write failed)
Failed to upload file: /tmp/498795789.itmsp/f43b28f31f34618cac4fe239c056fc057778874e3c78f80d291bd97a50d7e120.ipa

[23:30:40]: [iTMSTransporter] [2020-07-02 23:30:40 PDT] <main> DBG-X:   parameter ShouldUseRESTAPIs = false

[23:30:40]: [iTMSTransporter] [2020-07-02 23:30:40 PDT] <main> DBG-X:   parameter Success = true

[23:30:40]: [iTMSTransporter] [2020-07-02 23:30:40 PDT] <main> DBG-X:   parameter StreamingSettings = {LogStreamingEnabled=true, MultipartUploadsEnabled=true, AssetDescriptionStreamingEnabled=false}

[23:30:40]: [iTMSTransporter] [2020-07-02 23:30:40 PDT] <main>  INFO: Transporter's command line arguments are: -m upload -u <scrubbed> -p **hidden value** -f /tmp/498795789.itmsp -t DAV,Signiant -k 100000 -itc_provider RalphGooteeIII58973501

[23:30:40]: [iTMSTransporter] 

[23:30:40]: [iTMSTransporter] 

[23:30:40]: [iTMSTransporter] 

[23:30:40]: [iTMSTransporter] Package Summary:

[23:30:40]: [iTMSTransporter]  

[23:30:40]: [iTMSTransporter] 1 package(s) were not uploaded because they had problems:

[23:30:40]: [iTMSTransporter] 	/tmp/498795789.itmsp - Error Messages:

[23:30:40]: [iTMSTransporter] 		     An error occurred while uploading the file f43b28f31f34618cac4fe239c056fc057778874e3c78f80d291bd97a50d7e120.ipa.

[23:30:40]: [iTMSTransporter] 		An exception has occurred: Broken pipe (Write failed)

[23:30:40]: [iTMSTransporter] 		  Failed to upload file: /tmp/498795789.itmsp/f43b28f31f34618cac4fe239c056fc057778874e3c78f80d291bd97a50d7e120.ipa

[23:30:40]: [iTMSTransporter] [2020-07-02 23:30:40 PDT] <main> DBG-X: Returning 1

[23:30:40]: iTunes Transporter output above ^
[23:30:40]: An error occurred while uploading the file f43b28f31f34618cac4fe239c056fc057778874e3c78f80d291bd97a50d7e120.ipa.
An exception has occurred: Broken pipe (Write failed)
Failed to upload file: /tmp/498795789.itmsp/f43b28f31f34618cac4fe239c056fc057778874e3c78f80d291bd97a50d7e120.ipa
Return status of iTunes Transporter was 1: An error occurred while uploading the file f43b28f31f34618cac4fe239c056fc057778874e3c78f80d291bd97a50d7e120.ipa.
\nAn exception has occurred: Broken pipe (Write failed)
\nFailed to upload file: /tmp/498795789.itmsp/f43b28f31f34618cac4fe239c056fc057778874e3c78f80d291bd97a50d7e120.ipa
The call to the iTMSTransporter completed with a non-zero exit status: 1. This indicates a failure.
[23:30:40]: $ git rev-parse hotfix
[23:30:40]: ▸ bb122e2c94c2346f2f00c0449f63f2906604de64
[23:30:40]: $ git log -1 --format='%an' bb122e2c94c2346f2f00c0449f63f2906604de64~0
[23:30:40]: ▸ Ben Asher
[23:30:40]: $ git rev-parse bb122e2c94c2346f2f00c0449f63f2906604de64~0
[23:30:40]: ▸ bb122e2c94c2346f2f00c0449f63f2906604de64
[23:30:40]: $ git log -1 --format='%an' bb122e2c94c2346f2f00c0449f63f2906604de64
[23:30:40]: ▸ Ben Asher
[23:30:40]: $ git log -1 --format='%B' bb122e2c94c2346f2f00c0449f63f2906604de64
[23:30:40]: ▸ Upgrade fastlane again (#10282)
[23:30:40]: -----------------------------------
[23:30:40]: --- Sending Build Slack Message ---
[23:30:40]: -----------------------------------
+---------------+--------------------+
|            Lane Context            |
+---------------+--------------------+
| PLATFORM_NAME | ios                |
| LANE_NAME     | ios uploadplangrid |
+---------------+--------------------+
[23:30:40]: Could not upload binary to App Store Connect. Check out the error above

+------+---------------------+-------------+
|             fastlane summary             |
+------+---------------------+-------------+
| Step | Action              | Time (in s) |
+------+---------------------+-------------+
| 1    | unlock_keychain     | 0           |
| 💥   | upload_to_app_store | 639         |
+------+---------------------+-------------+

[23:30:40]: fastlane finished with errors

Looking for related GitHub issues on fastlane/fastlane...

➡️  [deliver] During Deliver process I get a Transporter error: Could not create the Java Virtual Machine
    https://github.com/fastlane/fastlane/issues/15323 [open] 33 💬
    2 days ago

➡️  upload_to_app_store with submit_for_review(true) returns "ITC.response.error.OPERATION_FAILED" 
    https://github.com/fastlane/fastlane/issues/16442 [open] 17 💬
    2 weeks ago

➡️  fastlane deliver: Potential server error received: 'Unexpected Error'
    https://github.com/fastlane/fastlane/issues/16621 [open] 120 💬
    2 days ago

and 108 more at: https://github.com/fastlane/fastlane/search?q=Could%20not%20upload%20binary%20to%20App%20Store%20Connect.%20Check%20out%20the%20error%20above&type=Issues&utf8=✓

🔗  You can ⌘ + double-click on links to open them directly in your browser.

[!] Could not upload binary to App Store Connect. Check out the error above
[Pipeline] echo
INCOMING CONFIG.STAGE: Upload build to iTunes
[Pipeline] ansiColor
[Pipeline] {
[Pipeline] echo
Upload build to iTunes was set to failure</pre>
</details>

### Environment
I filed #16748 becuase our env command is failing. I've included a bunch of info below. Please let me know if you need more!
- fastlane 2.150.1
- Ruby 2.6.5
- macOS 10.15.4
- Xcode 11.5

Gemfile.lock
<details><pre>
GEM
  remote: https://rubygems.org/
  specs:
    CFPropertyList (3.0.2)
    activesupport (4.2.11.3)
      i18n (~> 0.7)
      minitest (~> 5.1)
      thread_safe (~> 0.3, >= 0.3.4)
      tzinfo (~> 1.1)
    addressable (2.7.0)
      public_suffix (>= 2.0.2, < 5.0)
    algoliasearch (1.27.2)
      httpclient (~> 2.8, >= 2.8.3)
      json (>= 1.5.1)
    ast (2.4.0)
    atomos (0.1.3)
    aws-eventstream (1.1.0)
    aws-partitions (1.338.0)
    aws-sdk-core (3.103.0)
      aws-eventstream (~> 1, >= 1.0.2)
      aws-partitions (~> 1, >= 1.239.0)
      aws-sigv4 (~> 1.1)
      jmespath (~> 1.0)
    aws-sdk-kms (1.36.0)
      aws-sdk-core (~> 3, >= 3.99.0)
      aws-sigv4 (~> 1.1)
    aws-sdk-s3 (1.73.0)
      aws-sdk-core (~> 3, >= 3.102.1)
      aws-sdk-kms (~> 1)
      aws-sigv4 (~> 1.1)
    aws-sigv4 (1.2.1)
      aws-eventstream (~> 1, >= 1.0.2)
    babosa (1.0.3)
    claide (1.0.3)
    claide-plugins (0.9.2)
      cork
      nap
      open4 (~> 1.3)
    cocoapods (1.9.3)
      activesupport (>= 4.0.2, < 5)
      claide (>= 1.0.2, < 2.0)
      cocoapods-core (= 1.9.3)
      cocoapods-deintegrate (>= 1.0.3, < 2.0)
      cocoapods-downloader (>= 1.2.2, < 2.0)
      cocoapods-plugins (>= 1.0.0, < 2.0)
      cocoapods-search (>= 1.0.0, < 2.0)
      cocoapods-stats (>= 1.0.0, < 2.0)
      cocoapods-trunk (>= 1.4.0, < 2.0)
      cocoapods-try (>= 1.1.0, < 2.0)
      colored2 (~> 3.1)
      escape (~> 0.0.4)
      fourflusher (>= 2.3.0, < 3.0)
      gh_inspector (~> 1.0)
      molinillo (~> 0.6.6)
      nap (~> 1.0)
      ruby-macho (~> 1.4)
      xcodeproj (>= 1.14.0, < 2.0)
    cocoapods-core (1.9.3)
      activesupport (>= 4.0.2, < 6)
      algoliasearch (~> 1.0)
      concurrent-ruby (~> 1.1)
      fuzzy_match (~> 2.0.4)
      nap (~> 1.0)
      netrc (~> 0.11)
      typhoeus (~> 1.0)
    cocoapods-deintegrate (1.0.4)
    cocoapods-downloader (1.3.0)
    cocoapods-plugins (1.0.0)
      nap
    cocoapods-search (1.0.0)
    cocoapods-stats (1.1.0)
    cocoapods-trunk (1.5.0)
      nap (>= 0.8, < 2.0)
      netrc (~> 0.11)
    cocoapods-try (1.2.0)
    colored (1.2)
    colored2 (3.1.2)
    commander-fastlane (4.4.6)
      highline (~> 1.7.2)
    concurrent-ruby (1.1.6)
    cork (0.3.0)
      colored2 (~> 3.1)
    danger (6.0.9)
      claide (~> 1.0)
      claide-plugins (>= 0.9.2)
      colored2 (~> 3.1)
      cork (~> 0.1)
      faraday (~> 0.9)
      faraday-http-cache (~> 2.0)
      git (~> 1.5)
      kramdown (~> 2.0)
      kramdown-parser-gfm (~> 1.0)
      no_proxy_fix
      octokit (~> 4.7)
      terminal-table (~> 1)
    danger-junit (1.0.0)
      danger (> 2.0)
      ox (~> 2.0)
    danger-plugin-api (1.0.0)
      danger (> 2.0)
    danger-swiftlint (0.24.2)
      danger
      rake (> 10)
      thor (~> 0.19)
    danger-xcode_summary (0.5.1)
      danger-plugin-api (~> 1.0)
    declarative (0.0.20)
    declarative-option (0.1.0)
    digest-crc (0.6.1)
      rake (~> 13.0)
    domain_name (0.5.20190701)
      unf (>= 0.0.5, < 1.0.0)
    dotenv (2.7.5)
    emoji_regex (3.0.0)
    escape (0.0.4)
    ethon (0.12.0)
      ffi (>= 1.3.0)
    excon (0.75.0)
    faraday (0.17.3)
      multipart-post (>= 1.2, < 3)
    faraday-cookie_jar (0.0.6)
      faraday (>= 0.7.4)
      http-cookie (~> 1.0.0)
    faraday-http-cache (2.2.0)
      faraday (>= 0.8)
    faraday_middleware (0.14.0)
      faraday (>= 0.7.4, < 1.0)
    fastimage (2.1.7)
    fastlane (2.150.1)
      CFPropertyList (>= 2.3, < 4.0.0)
      addressable (>= 2.3, < 3.0.0)
      aws-sdk-s3 (~> 1.0)
      babosa (>= 1.0.2, < 2.0.0)
      bundler (>= 1.12.0, < 3.0.0)
      colored
      commander-fastlane (>= 4.4.6, < 5.0.0)
      dotenv (>= 2.1.1, < 3.0.0)
      emoji_regex (>= 0.1, < 4.0)
      excon (>= 0.71.0, < 1.0.0)
      faraday (>= 0.17, < 2.0)
      faraday-cookie_jar (~> 0.0.6)
      faraday_middleware (>= 0.13.1, < 2.0)
      fastimage (>= 2.1.0, < 3.0.0)
      gh_inspector (>= 1.1.2, < 2.0.0)
      google-api-client (>= 0.37.0, < 0.39.0)
      google-cloud-storage (>= 1.15.0, < 2.0.0)
      highline (>= 1.7.2, < 2.0.0)
      json (< 3.0.0)
      jwt (~> 2.1.0)
      mini_magick (>= 4.9.4, < 5.0.0)
      multi_xml (~> 0.5)
      multipart-post (~> 2.0.0)
      plist (>= 3.1.0, < 4.0.0)
      rubyzip (>= 2.0.0, < 3.0.0)
      security (= 0.1.3)
      simctl (~> 1.6.3)
      slack-notifier (>= 2.0.0, < 3.0.0)
      terminal-notifier (>= 2.0.0, < 3.0.0)
      terminal-table (>= 1.4.5, < 2.0.0)
      tty-screen (>= 0.6.3, < 1.0.0)
      tty-spinner (>= 0.8.0, < 1.0.0)
      word_wrap (~> 1.0.0)
      xcodeproj (>= 1.13.0, < 2.0.0)
      xcpretty (~> 0.3.0)
      xcpretty-travis-formatter (>= 0.0.3)
    fastlane-plugin-appcenter (1.8.0)
    fastlane-plugin-bugsnag (1.4.2)
      git
      xml-simple
    fastlane-plugin-trainer (0.4.1)
      trainer (>= 0.7.0)
    ffi (1.13.0)
    fourflusher (2.3.1)
    fuzzy_match (2.0.4)
    gh_inspector (1.1.3)
    gli (2.19.1)
    google-api-client (0.38.0)
      addressable (~> 2.5, >= 2.5.1)
      googleauth (~> 0.9)
      httpclient (>= 2.8.1, < 3.0)
      mini_mime (~> 1.0)
      representable (~> 3.0)
      retriable (>= 2.0, < 4.0)
      signet (~> 0.12)
    google-cloud-core (1.5.0)
      google-cloud-env (~> 1.0)
      google-cloud-errors (~> 1.0)
    google-cloud-env (1.3.2)
      faraday (>= 0.17.3, < 2.0)
    google-cloud-errors (1.0.1)
    google-cloud-storage (1.26.2)
      addressable (~> 2.5)
      digest-crc (~> 0.4)
      google-api-client (~> 0.33)
      google-cloud-core (~> 1.2)
      googleauth (~> 0.9)
      mini_mime (~> 1.0)
    googleauth (0.13.0)
      faraday (>= 0.17.3, < 2.0)
      jwt (>= 1.4, < 3.0)
      memoist (~> 0.16)
      multi_json (~> 1.11)
      os (>= 0.9, < 2.0)
      signet (~> 0.14)
    hashie (4.1.0)
    highline (1.7.10)
    http-cookie (1.0.3)
      domain_name (~> 0.5)
    httpclient (2.8.3)
    i18n (0.9.5)
      concurrent-ruby (~> 1.0)
    jaro_winkler (1.5.4)
    jira-ruby (1.6.0)
      activesupport
      multipart-post
      oauth (~> 0.5, >= 0.5.0)
    jmespath (1.4.0)
    json (2.3.1)
    jwt (2.1.0)
    kramdown (2.2.1)
      rexml
    kramdown-parser-gfm (1.1.0)
      kramdown (~> 2.0)
    memoist (0.16.2)
    mime-types (3.3.1)
      mime-types-data (~> 3.2015)
    mime-types-data (3.2020.0512)
    mini_magick (4.10.1)
    mini_mime (1.0.2)
    mini_portile2 (2.4.0)
    minitest (5.14.1)
    molinillo (0.6.6)
    multi_json (1.14.1)
    multi_xml (0.6.0)
    multipart-post (2.0.0)
    nanaimo (0.2.6)
    nap (1.1.0)
    naturally (2.2.0)
    netrc (0.11.0)
    no_proxy_fix (0.1.2)
    nokogiri (1.10.9)
      mini_portile2 (~> 2.4.0)
    oauth (0.5.4)
    octokit (4.18.0)
      faraday (>= 0.9)
      sawyer (~> 0.8.0, >= 0.5.3)
    open4 (1.3.4)
    os (1.1.0)
    ox (2.13.2)
    pager_duty-connection (1.1.0)
      activesupport (>= 3.2, < 6.0)
      faraday (~> 0.8, < 1.0)
      faraday_middleware (~> 0.8, < 1.0)
      hashie (>= 1.2)
    parallel (1.19.1)
    parser (2.7.1.2)
      ast (~> 2.4.0)
    phraseapp-ruby (3.2.0)
    plist (3.5.0)
    powerpack (0.1.2)
    psych (2.2.4)
    public_suffix (4.0.5)
    rainbow (3.0.0)
    rake (13.0.1)
    rchardet (1.8.0)
    representable (3.0.4)
      declarative (< 0.1.0)
      declarative-option (< 0.2.0)
      uber (< 0.2.0)
    retriable (3.1.2)
    rexml (3.2.4)
    rouge (2.0.7)
    rubocop (0.64.0)
      jaro_winkler (~> 1.5.1)
      parallel (~> 1.10)
      parser (>= 2.5, != 2.5.1.1)
      powerpack (~> 0.1)
      rainbow (>= 2.2.2, < 4.0)
      ruby-progressbar (~> 1.7)
      unicode-display_width (~> 1.4.0)
    ruby-macho (1.4.0)
    ruby-progressbar (1.10.1)
    rubyzip (2.3.0)
    sawyer (0.8.2)
      addressable (>= 2.3.5)
      faraday (> 0.8, < 2.0)
    security (0.1.3)
    signet (0.14.0)
      addressable (~> 2.3)
      faraday (>= 0.17.3, < 2.0)
      jwt (>= 1.5, < 3.0)
      multi_json (~> 1.10)
    simctl (1.6.8)
      CFPropertyList
      naturally
    slack-notifier (2.3.2)
    slack-ruby-client (0.14.6)
      activesupport
      faraday (>= 0.9)
      faraday_middleware
      gli
      hashie
      websocket-driver
    sorbet (0.4.4755)
      sorbet-static (= 0.4.4755)
    sorbet-runtime (0.4.4755)
    sorbet-static (0.4.4755-universal-darwin-14)
    terminal-notifier (2.0.0)
    terminal-table (1.8.0)
      unicode-display_width (~> 1.1, >= 1.1.1)
    thor (0.20.3)
    thread_safe (0.3.6)
    trainer (0.9.1)
      fastlane (>= 2.25.0)
      plist (>= 3.1.0, < 4.0.0)
    tty-cursor (0.7.1)
    tty-screen (0.8.0)
    tty-spinner (0.9.3)
      tty-cursor (~> 0.7)
    typhoeus (1.4.0)
      ethon (>= 0.9.0)
    tzinfo (1.2.7)
      thread_safe (~> 0.1)
    uber (0.1.0)
    unf (0.1.4)
      unf_ext
    unf_ext (0.0.7.7)
    unicode-display_width (1.4.1)
    websocket-driver (0.7.2)
      websocket-extensions (>= 0.1.0)
    websocket-extensions (0.1.5)
    word_wrap (1.0.0)
    xcodeproj (1.17.0)
      CFPropertyList (>= 2.3.3, < 4.0)
      atomos (~> 0.1.3)
      claide (>= 1.0.2, < 2.0)
      colored2 (~> 3.1)
      nanaimo (~> 0.2.6)
    xcpretty (0.3.0)
      rouge (~> 2.0.7)
    xcpretty-json-formatter (0.1.1)
      xcpretty (~> 0.2, >= 0.0.7)
    xcpretty-travis-formatter (1.0.0)
      xcpretty (~> 0.2, >= 0.0.7)
    xml-simple (1.1.5)

PLATFORMS
  ruby

DEPENDENCIES
  automation_library!
  cocoapods (~> 1.9)
  danger (~> 6.0)
  danger-junit
  danger-swiftlint
  danger-xcode_summary
  fastlane (~> 2.150)
  fastlane-plugin-appcenter
  fastlane-plugin-automation_library!
  fastlane-plugin-bugsnag
  fastlane-plugin-trainer
  git!
  mime-types
  netrc
  nokogiri
  rubocop (= 0.64.0)
  sorbet (= 0.4.4755)
  sorbet-runtime (= 0.4.4755)
  xcpretty-json-formatter

BUNDLED WITH
   2.1.4
</pre></details>

**Comment by benasher44** [comment]
Yes sorry fastlane-bot see #16748 😬

**Comment by pfay-fc** [comment]
Same issue here - I upgraded to 2.150.1 after I'd been running with one of the 2.150.0 rc builds and now I'm consistently getting a transporter broken pipe when uploading the IPA since I updated yesterday. I had initially thought it might just be an intermittent issue on apple's end per https://github.com/fastlane/fastlane/issues/16462 but that doesn't seem to be the case this time.

**Comment by matcartmill** [comment]
I'm seeing the same issue, but was seeing it first on 2.149.1 so I updated to 2.150.1 to see if that would help, but it didn't. I believe this is more like an iTunes server issue but will try to push a TestFlight build from Xcode just to be sure. Will post back findings.

**Comment by Lin-Bing** [comment]
I got the same error with fastlane version 2.150.0...help...


================================== fastlane env ===============================

### Stack

| Key                         | Value                                       |
| --------------------------- | ------------------------------------------- |
| OS                          | 10.15.2                                     |
| Ruby                        | 2.4.0                                       |
| Bundler?                    | false                                       |
| Git                         | git version 2.21.0 (Apple Git-122.2)        |
| Installation Source         | /usr/local/bin/fastlane                     |
| Host                        | Mac OS X 10.15.2 (19C57)                    |
| Ruby Lib Dir                | /usr/local/lib                              |
| OpenSSL Version             | OpenSSL 1.0.2j  26 Sep 2016                 |
| Is contained                | false                                       |
| Is homebrew                 | false                                       |
| Is installed via Fabric.app | false                                       |
| Xcode Path                  | /Applications/Xcode.app/Contents/Developer/ |
| Xcode Version               | 11.3                                        |


### System Locale

| Variable | Value       |   |
| -------- | ----------- | - |
| LANG     | zh_CN.UTF-8 | ✅ |
| LC_ALL   |             |   |
| LANGUAGE |             |   |



==================================== error log =========================================


[09:30:09]: Loading './fastlane/metadata/review_information/email_address.txt'...
[09:30:09]: Loading './fastlane/metadata/review_information/demo_user.txt'...
[09:30:09]: Loading './fastlane/metadata/review_information/demo_password.txt'...
[09:30:09]: Loading './fastlane/metadata/review_information/notes.txt'...
[09:30:39]: Will begin uploading metadata for '5.9.992' on App Store Connect
[09:30:39]: Uploading metadata to App Store Connect for localized version 'zh-Hans'
[09:30:40]: Uploading metadata to App Store Connect for localized version 'en-GB'
[09:31:01]: Uploading metadata to App Store Connect for localized info 'en-GB'
[09:31:04]: Uploading metadata to App Store Connect for localized info 'zh-Hans'
[09:31:07]: Uploading metadata to App Store Connect for version
[09:31:09]: Category 'MZGenre.Travel' from iTunesConnect as been deprecated. Please replace with 'TRAVEL'
[09:31:09]: Category 'MZGenre.Lifestyle' from iTunesConnect as been deprecated. Please replace with 'LIFESTYLE'
[09:31:09]: You can find more info at https://docs.fastlane.tools/actions/deliver/#reference
[09:31:18]: Uploading app review information to App Store Connect
[09:31:22]: Uploading binary to App Store Connect
[09:31:23]: Going to upload updated app to App Store Connect
[09:31:23]: This might take a few minutes. Please don't interrupt the script.
[09:33:21]: [Transporter Error Output]: An error occurred while uploading the file 727ff3e24d8f44c5c29f3a4698d122ad5821718f45cab45d6890d831b3126b6c.ipa.
[09:33:21]: [Transporter Error Output]: An exception has occurred: Broken pipe (Write failed)
[09:33:21]: [Transporter Error Output]: Failed to upload file: /tmp/509256354.itmsp/727ff3e24d8f44c5c29f3a4698d122ad5821718f45cab45d6890d831b3126b6c.ipa
[09:33:26]: Transporter transfer failed.
[09:33:26]: 
[09:33:26]: An error occurred while uploading the file 727ff3e24d8f44c5c29f3a4698d122ad5821718f45cab45d6890d831b3126b6c.ipa.
An exception has occurred: Broken pipe (Write failed)
Failed to upload file: /tmp/509256354.itmsp/727ff3e24d8f44c5c29f3a4698d122ad5821718f45cab45d6890d831b3126b6c.ipa

**Comment by matcartmill** [comment]
Was able to upload via Xcode without issue.

**Comment by Lin-Bing** [comment]
> Was able to upload via Xcode without issue.

Finally，I upload successfully via 'Transporter App'

**Comment by camilape** [comment]
Same issue here with Fastlane version 2.148.1., I've been seeing this kind of issue since Fasltane's version 2.149.0 but then I did a downgrade to v 2.148.1 but is still happening. I don't know if this is an Apple's API server error but it's weird because using Xcode there is no problem. I'd be grateful if someone can help.

**Comment by AnarionII** [comment]
Same issue, I tried with fastlane 2.150.1 and 2.149.1.

**Comment by jchown** [comment]
I used the Signiant delivery protocol mentioned here on a similar issue: https://github.com/fastlane/fastlane/issues/13572#issuecomment-431898094 and I managed to get a build uploaded using 2.145.0

**Comment by marcoruggine** [comment]
Same issue here with 2.150.2, it has worked perfectly up to July 2nd but now it just fails.

**Comment by ir2pid** [comment]
Same issue for us, previous fastlane versions dont work as well.

**Comment by andre-alves-ifood** [comment]
Signiant delivery protocol worked for us!

**Comment by Tnk-Yuki** [comment]
Maybe it's the DAV delivery protocol settings that cause it to fail.

https://help.apple.com/itc/transporteruserguide/en.lproj/static.html#apdATD1E112-D1E1A1303-D1E112A1126

> Because Apple periodically releases new versions and updates to its software, servers listed in the table might change mid-release.

It seems that it is recommended not to set the delivery protocol.
https://help.apple.com/itc/transporteruserguide/#/apdATD1E1288-D1E1A1303-D1E1288A1126

**Comment by scottlemke** [comment]
> Signiant delivery protocol worked for us!

Thank you for this! We're able to use this as a workaround for now since this has been holding up our release pipeline for a couple days.

**Comment by JimHowell20** [comment]
The Signiant protocol is blocked by a lot of corporate firewalls, so this workaround won't be an option for some people.  Hopefully there will be a fix for this soon.

**Comment by scottlemke** [comment]
> The Signiant protocol is blocked by a lot of corporate firewalls, so this workaround won't be an option for some people. Hopefully there will be a fix for this soon.

Agree. We're using Bitrise, and it worked out OK. Hopefully an official fix is in the works.

**Comment by mohsinulhaq** [comment]
@scottlemke how did you use Signiant protocol in Bitrise?

**Comment by scottlemke** [comment]
> @scottlemke how did you use Signiant protocol in Bitrise?

Create an environment variable called `DELIVER_ITMSTRANSPORTER_ADDITIONAL_UPLOAD_PARAMETERS` and set it to `-t Signiant`.

**Comment by ir2pid** [comment]
Signiant doesnt work for us, giving a connection issue.
```
 [03:57:09]: Going to upload updated app to App Store Connect
 [03:57:09]: This might take a few minutes. Please don't interrupt the script.
 [04:06:31]: Transporter transfer failed.
 [04:06:31]: 
 [04:06:31]: unable to verify WebDAV transport: The target server failed to respond
 An error occurred while uploading the package: xx.itmsp.
 [04:06:31]: [iTMSTransporter] [2020-07-05 04:06:31 CEST] <main> DBG-X: Setting the target rate to 15000001 bytes per second.
 [04:06:31]: [iTMSTransporter] [2020-07-05 04:06:31 CEST] <main> DBG-X: Setting the throttle to 12500000 bytes per second.
 [04:06:31]: [iTMSTransporter] Exception in thread "Transport Protocol Seeker" java.lang.ArrayIndexOutOfBoundsException: 1
 [04:06:31]: [iTMSTransporter] 	at com.signiant.interactivetransfer.engine.TransferEngine$5.run(TransferEngine.java:3603)
 [04:06:31]: [iTMSTransporter] 
 [04:06:31]: [iTMSTransporter] 
 [04:06:31]: [iTMSTransporter] 
 [04:06:31]: [iTMSTransporter] Package Summary:
 [04:06:31]: [iTMSTransporter]  
 [04:06:31]: [iTMSTransporter] 1 package(s) were not uploaded because they had problems:
 [04:06:31]: [iTMSTransporter] 	/var/folders/7g/88jlglrx7jxds9qk353rl93w0000gp/T/xx-xx-ghdbne/xx.itmsp - Error Messages:
 [04:06:31]: [iTMSTransporter] 		unable to verify WebDAV transport: The target server failed to respond
 [04:06:31]: [iTMSTransporter] 		An error occurred while uploading the package: xx.itmsp.
 [04:06:31]: [iTMSTransporter] 		An exception has occurred: The target server failed to respond
 [04:06:31]: [iTMSTransporter] [2020-07-05 04:06:31 CEST] <main> DBG-X: Returning 1
 [04:06:31]: iTunes Transporter output above ^
 [04:06:31]: unable to verify WebDAV transport: The target server failed to respond
 An error occurred while uploading the package: xx.itmsp.
 An exception has occurred: The target server failed to respond
 \nAn exception has occurred: The target server failed to respond
 The call to the iTMSTransporter completed with a non-zero exit status: 1. This indicates a failure.
 [04:06:31]: upload failed! Error uploading ipa file, for more information see above

```

**Comment by sneko** [comment]
Same with `fastlane 2.141.0` 😞

**Comment by castus** [comment]
@janpio For me this issue occurs for `upload_to_testflight` action, so it is more related to `pilot` tool, not `deliver`. Am I correct? :)

**Comment by ajaykn** [comment]
@castus I think we can use the same flag `DELIVER_ITMSTRANSPORTER_ADDITIONAL_UPLOAD_PARAMETERS ` for uploading to TestFlight as well (which is pilot tool too), as per the docs [here](https://github.com/fastlane/fastlane/blob/master/fastlane/lib/fastlane/actions/docs/upload_to_testflight.md#firewall-issues)

**Comment by taisukeh** [comment]
Set that variable to blank, it works well in our environment.
```
ENV['DELIVER_ITMSTRANSPORTER_ADDITIONAL_UPLOAD_PARAMETERS'] = " "
```

**Comment by AnarionII** [comment]
> Set that variable to blank, it works well in our environment.
> 
> ```
> ENV['DELIVER_ITMSTRANSPORTER_ADDITIONAL_UPLOAD_PARAMETERS'] = " "
> ```

Thank you Taisukeh, it works!

**Comment by castus** [comment]
Ok, so if make it blank works, it means that [workaround about transporter](https://github.com/fastlane/fastlane/blob/1724017c0a39a864001c13db8d054bd580589a7e/fastlane_core/lib/fastlane_core/itunes_transporter.rb#L151) is not needed anymore. It would also be aligned with [Apple's Transporter User Guide 2.0 documentation](https://help.apple.com/itc/transporteruserguide/#/apdATD1E1288-D1E1A1303-D1E1288A1126). I can make a PR with that, what do you think guys?

**Comment by daniel-toth-leeder** [comment]
> Ok, so if make it blank works, it means that [workaround about transporter](https://github.com/fastlane/fastlane/blob/1724017c0a39a864001c13db8d054bd580589a7e/fastlane_core/lib/fastlane_core/itunes_transporter.rb#L151) is not needed anymore. It would also be aligned with [Apple's Transporter User Guide 2.0 documentation](https://help.apple.com/itc/transporteruserguide/#/apdATD1E1288-D1E1A1303-D1E1288A1126). I can make a PR with that, what do you think guys?

I can confirm, for us making it contain 1 space as @taisukeh mentioned it, solves the issue on our side as well. Looking forward for the PR :)

**Comment by rickmanelius** [comment]
I can confirm that this unblocked me for Bitrise.io by making the following changes to my bitrise.yml.

```
    - deploy-to-itunesconnect-deliver:
        inputs:
        - fastlane_version: 2.150.0
        - itms_upload_parameters: "-t Signiant"
        - options: ' '
```

Note that I don't think I still need to pin the version or send the space in the options. I think the one that solved this was the `itms_upload_parameters` entry. I'm just leaving the others for a bit until things shake out. I may suggest changing the default upstream to the Bitrise.io step library team.

**Comment by mdstroebel** [comment]
If you are using Bitrise, you can also set the Delivery method in the "Deploy to iTunes Connect" step's "Debug" section:

![image](https://user-images.githubusercontent.com/4019731/86671896-719ced00-c039-11ea-8b98-c724adb68098.png)

I believe you can comma separate the value to allow for failover if required.

Setting it to Signiant worked in our case.

**Comment by pepusz** [comment]
> If you are using Bitrise, you can also set the Delivery method in the "Deploy to iTunes Connect" step's "Debug" section:
> 
> ![image](https://user-images.githubusercontent.com/4019731/86671896-719ced00-c039-11ea-8b98-c724adb68098.png)
> 
> I believe you can comma separate the value to allow for failover if required.
> 
> Setting it to Signiant worked in our case.

Yepp this is working in our case too.

**Comment by marcoruggine** [comment]
I can confirm that -t Signiant in Bitrise is solving my problem as well

**Comment by TheWirv** [comment]
> 
> 
> Set that variable to one white space, it works well in our environment.
> 
> ```
> ENV['DELIVER_ITMSTRANSPORTER_ADDITIONAL_UPLOAD_PARAMETERS'] = " "
> ```

This worked for us perfectly, after multiple unsuccessful tries to deploy. Signiant doesn't work for is, it can't establish a connection. I guess it is blocked in our firewall as @JimHowell20 has already pointed out.

**Comment by joshdholtz** [comment]
👋 The fix for this will be released in `2.151.0` later this evening!

**Comment by iiAtlas** [comment]
<details><summary>✅ fastlane environment ✅</summary>

### Stack

| Key                         | Value                                                          |
| --------------------------- | -------------------------------------------------------------- |
| OS                          | 10.15.5                                                        |
| Ruby                        | 2.6.3                                                          |
| Bundler?                    | false                                                          |
| Git                         | git version 2.24.3 (Apple Git-128)                             |
| Installation Source         | /usr/local/bin/fastlane                                        |
| Host                        | Mac OS X 10.15.5 (19F101)                                      |
| Ruby Lib Dir                | /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib |
| OpenSSL Version             | LibreSSL 2.8.3                                                 |
| Is contained                | false                                                          |
| Is homebrew                 | false                                                          |
| Is installed via Fabric.app | false                                                          |
| Xcode Path                  | /Applications/Xcode.app/Contents/Developer/                    |
| Xcode Version               | 11.5                                                           |


### System Locale

| Variable | Value       |   |
| -------- | ----------- | - |
| LANG     | en_US.UTF-8 | ✅ |
| LC_ALL   |             |   |
| LANGUAGE |             |   |


### fastlane files:

<details><summary>`./fastlane/Fastfile`</summary>

```ruby
# This file contains the fastlane.tools configuration
# You can find the documentation at https://docs.fastlane.tools
#
# For a list of all available actions, check out
#
#     https://docs.fastlane.tools/actions
#
# For a list of all available plugins, check out
#
#     https://docs.fastlane.tools/plugins/available-plugins
#

# Uncomment the line if you want fastlane to automatically update itself
before_all do
  update_fastlane
end

default_platform(:ios)

platform :ios do
  desc "Push a new beta build to TestFlight"
  lane :beta do
    match(type: "appstore")

    # increment_version_number(bump_type: "minor")
    increment_build_number(xcodeproj: "tkt_react.xcodeproj")
    build_app(workspace: "tkt_react.xcworkspace", scheme: "tkt_react")

    upload_to_testflight(
      groups: ['Team']
    )
  end
end

```
</details>

<details><summary>`./fastlane/Appfile`</summary>

```ruby
app_identifier("com.tikkit.tkt-react") # The bundle identifier of your app
apple_id("atlas@tikkit.io") # Your Apple email address

itc_team_id("117926984") # App Store Connect Team ID
team_id("ZP8G2NEB87") # Developer Portal Team ID

# For more information about the Appfile, see:
#     https://docs.fastlane.tools/advanced/#appfile

```
</details>

### fastlane gems

| Gem      | Version | Update-Status |
| -------- | ------- | ------------- |
| fastlane | 2.152.0 | ✅ Up-To-Date  |


### Loaded fastlane plugins:

**No plugins Loaded**


<details><summary><b>Loaded gems</b></summary>

| Gem                       | Version      |
| ------------------------- | ------------ |
| did_you_mean              | 1.3.0        |
| slack-notifier            | 2.3.2        |
| rouge                     | 2.0.7        |
| xcpretty                  | 0.3.0        |
| terminal-notifier         | 2.0.0        |
| unicode-display_width     | 1.7.0        |
| terminal-table            | 1.8.0        |
| plist                     | 3.5.0        |
| addressable               | 2.7.0        |
| multipart-post            | 2.0.0        |
| word_wrap                 | 1.0.0        |
| tty-cursor                | 0.7.1        |
| tty-spinner               | 0.9.3        |
| babosa                    | 1.0.3        |
| colored                   | 1.2          |
| highline                  | 1.7.10       |
| commander-fastlane        | 4.4.6        |
| unf_ext                   | 0.0.7.7      |
| unf                       | 0.1.4        |
| domain_name               | 0.5.20190701 |
| http-cookie               | 1.0.3        |
| faraday-cookie_jar        | 0.0.6        |
| faraday                   | 0.17.3       |
| fastimage                 | 2.1.7        |
| gh_inspector              | 1.1.3        |
| mini_magick               | 4.10.1       |
| rubyzip                   | 2.3.0        |
| security                  | 0.1.3        |
| xcpretty-travis-formatter | 1.0.0        |
| bundler                   | 2.1.4        |
| naturally                 | 2.2.0        |
| simctl                    | 1.6.8        |
| uber                      | 0.1.0        |
| declarative-option        | 0.1.0        |
| representable             | 3.0.4        |
| retriable                 | 3.1.2        |
| mini_mime                 | 1.0.2        |
| signet                    | 0.14.0       |
| httpclient                | 2.8.3        |
| google-api-client         | 0.38.0       |
| nanaimo                   | 0.2.6        |
| colored2                  | 3.1.2        |
| claide                    | 1.0.3        |
| CFPropertyList            | 3.0.2        |
| atomos                    | 0.1.3        |
| xcodeproj                 | 1.17.0       |
| public_suffix             | 4.0.5        |
| tty-screen                | 0.8.0        |
| excon                     | 0.75.0       |
| faraday_middleware        | 0.14.0       |
| json                      | 2.3.1        |
| dotenv                    | 2.7.6        |
| jwt                       | 2.2.1        |
| declarative               | 0.0.20       |
| multi_json                | 1.15.0       |
| os                        | 1.1.0        |
| memoist                   | 0.16.2       |
| googleauth                | 0.13.0       |
| rake                      | 13.0.1       |
| digest-crc                | 0.6.1        |
| google-cloud-errors       | 1.0.1        |
| google-cloud-env          | 1.3.3        |
| google-cloud-core         | 1.5.0        |
| google-cloud-storage      | 1.26.2       |
| emoji_regex               | 3.0.0        |
| aws-eventstream           | 1.1.0        |
| aws-sigv4                 | 1.2.1        |
| aws-partitions            | 1.341.0      |
| jmespath                  | 1.4.0        |
| aws-sdk-core              | 3.103.0      |
| aws-sdk-kms               | 1.36.0       |
| aws-sdk-s3                | 1.74.0       |
| forwardable               | 1.2.0        |
| logger                    | 1.3.0        |
| stringio                  | 0.0.2        |
| ipaddr                    | 1.2.2        |
| openssl                   | 2.1.2        |
| ostruct                   | 0.1.0        |
| strscan                   | 1.0.0        |
| date                      | 2.0.0        |
| fileutils                 | 1.1.0        |
| etc                       | 1.0.1        |
| io-console                | 0.4.7        |
| zlib                      | 1.0.0        |
| libxml-ruby               | 3.1.0        |
| rexml                     | 3.1.9        |
| psych                     | 3.1.0        |
| mutex_m                   | 0.1.0        |
| webrick                   | 1.4.2        |
</details>


*generated on:* **2020-07-14**
</details>

Hello! Sadly reporting broken pipe / write failed today on Fastlane `2.152.0`

<img width="1779" alt="Screen Shot 2020-07-14 at 10 46 38 AM" src="https://user-images.githubusercontent.com/764612/87440014-71fc3180-c5bf-11ea-85ef-792903083a7e.png">

With `ENV['DELIVER_ITMSTRANSPORTER_ADDITIONAL_UPLOAD_PARAMETERS'] = " "`

<img width="1792" alt="Screen Shot 2020-07-14 at 11 11 21 AM" src="https://user-images.githubusercontent.com/764612/87443412-732f5d80-c5c3-11ea-8308-d08776c560a1.png">

With `ENV["DELIVER_ITMSTRANSPORTER_ADDITIONAL_UPLOAD_PARAMETERS"] = "-t Signiant"`

<img width="1050" alt="Screen Shot 2020-07-14 at 12 49 39 PM" src="https://user-images.githubusercontent.com/764612/87453450-809f1480-c5d0-11ea-866a-11eda1d53983.png">

Any thoughts greatly appreciated!

**Comment by joshdholtz** [comment]
@iiAtlas Hey! Can you try not setting anything? It _should_ be smart enough to pick its own now

**Comment by iiAtlas** [comment]
Hey @joshdholtz thanks for the speedy reply.  Sorry about that, the first output above (broken pipe) resulted with no environment variable set.

**Comment by joshdholtz** [comment]
@iiAtlas Thanks for clarifying! Can you also try ENV["DELIVER_ITMSTRANSPORTER_ADDITIONAL_UPLOAD_PARAMETERS"] = "-t Aspera"? I don't know if that will help at all but this is a bit confusing 🤷‍♂️

**Comment by iiAtlas** [comment]
Sorry for the delay @joshdholtz -- that one is a no go!!

<img width="1764" alt="Screen Shot 2020-07-14 at 2 56 25 PM" src="https://user-images.githubusercontent.com/764612/87465298-679f5f00-c5e2-11ea-89ee-09a6482c02d7.png">

I'm also not pulling up any system issues when visiting [https://www.apple.com/support/systemstatus/](url) or [https://developer.apple.com/system-status/](url).  Not sure if that is a clue or just further confusion...

And to be clear, this is a working Fastlane environment for the past 8 months or so!  Thanks for the help.

**Comment by joshdholtz** [comment]
@iiAtlas Hmmmm 🤔  I've tried all of the different options on my end and things seem to be working fine for me 😔 Are you on a different network or a behind a firewall of some sort today?

**Comment by iiAtlas** [comment]
@joshdholtz I am on a different network than my usual but nothing that would have immediately caused concern!  I'll head to another spot and see if I have better results...  If it does prove to be network related then I am sorry for wasting your time!!!

**Comment by joshdholtz** [comment]
@iiAtlas Don't worry about my time 😇 This is what I'm here for!

**Comment by iiAtlas** [comment]
Total success! 🤦

Now to dig in to whats stopping this from working at my home base.... thanks @joshdholtz and sorry for dragging up the thread!  The mirrored error message mislead me....

**Comment by Skoti** [comment]
so the fix for this issue in #16774 - is actually the workaround for #13718
it's good to know the reason though 😄  thanks!
> Apple recommends to not specify transport option and let the tool to determine the best transport mode for packages.
